proto/expansion.proto

```diff
@@ -18,9 +18,10 @@ message Expansion {
   repeated uint32 costs = 1 [packed=true];
   repeated uint32 durations = 2 [packed=true];
   repeated uint32 distances = 3 [packed=true];
-  repeated EdgeStatus edge_status = 4;
+  repeated EdgeStatus edge_status = 4 [packed=true];
   repeated uint32 edge_id = 5 [packed=true];
   repeated uint32 pred_edge_id = 6 [packed=true];
+  repeated uint32 edge_status_flags = 7 [packed=true];
 
-  repeated Geometry geometries = 7;
+  repeated Geometry geometries = 8;
```

Comment:
moving things around in proto is very much a breaking change again it doesnt matter because we call this service beta but we should try not to do this when we dont need to

Reply:
You're right of course, but since we're breaking it anyway, I've moved it around ... well just for aesthetic reasons tbh.